### ISSUE TITLE
RoadMap

### ISSUE THREAD

**Issue by ldemk** [modelica/DCPLib#55]
Dear DCPLib Team

How mature is the DCLLib with respect to industrial usage in production?

Is there a roadmap with DCPLib development timeline?

Where can I find the documentation for the DCPLib usage?

**Comment by clemensschiffer** [comment]
Hello,

the DCPLib was developed as a reference implementation of the DCP standard as such it was released feature complete[^1] to be used and supported by the community.

For usage see the included example and the [wiki](https://github.com/modelica/DCPLib/wiki).
To get started build the examples and modify them, easiest on linux.
I have used the DCPLib for some use cases and corrected some minor bugs, as have others, see the PR and dev branch.

As with any open source project it comes 'as is' with no warranty.
If someone wants to use it in a commercial product they have to check if it meets their requirements.

For a roadmap: There is no new development to be done, only maintenance and improvements, as all the features of the protocol are there. Except for more transport protocols[^1].
I also have some tests to integrate here.

Kind regards
Clemens

[^1]: Transport protocols UDP/TCP are implemented, drivers for USB/Bluetooth/CAN are not.
